integ-test/src/test/java/org/opensearch/sql/legacy/SQLIntegTestCase.java

```diff
@@ -392,7 +392,7 @@ protected static JSONObject updateClusterSettings(ClusterSetting setting, RestCl
     return new JSONObject(executeRequest(request, client));
   }
 
-  protected static JSONObject updateClusterSettings(ClusterSetting setting) throws IOException {
+  public static JSONObject updateClusterSettings(ClusterSetting setting) throws IOException {
```

Comment:
why does this need to be made public?  It's being called by ITs that extend this class.

Reply:
I use it in correctness test, which does not inherit this class.